examples/01-sgp4xp/01-basic.py

```diff
@@ -0,0 +1,25 @@
+"""
+SGP4-XP Basic Usage
+===================
+"""
+
+from pysaal.elements import TLE
+from pysaal.time import Epoch
+
+# %%
+# Let's initialize a TLE object and a propagation epoch
+
+l1 = '1 61429U          24362.90619720 +.00000000  00000 0  55480 0 4 0001'
+l2 = '2 61429  88.9747 334.1365 0159713  22.5177  31.9343 14.3545506800001'
+
+tle = TLE().from_lines(l1, l2)
```

Comment:
As a classmethod, this can be instantiated as `TLE.from_lines(l1, l2)`

Reply:
done, same for other cases.